### PR TITLE
Add libusbx to msvc build

### PR DESCRIPTION
**Pull request by anosach-luxoft**
- Change cmake files to include libusbx headers
- Build libusbx static library

Libusbx library should be built with /W2 option using msvc because of libusbx internal warnings

### CODE REVIEW

**Comment by anosach-luxoft** [comment]
@nk0leg, @AGaliuzov, @pestOO, please review

**Comment by AGaliuzov** [comment]
Reviewed

File: src/3rd_party-static/libusbx-1.0.16/CMakeLists.txt [hunk 1]
```diff
@@ -10,21 +10,37 @@ include_directories (
...
+    ./libusb/descriptor.c
+    ./libusb/hotplug.c
+    ./libusb/io.c
+    ./libusb/strerror.c
+    ./libusb/sync.c
+    ./libusb/os/poll_windows.c
+    ./libusb/os/threads_windows.c
+    ./libusb/os/windows_usb.c
+  )
+else()
```
Comment (pestOO) [inline src/3rd_party-static/libusbx-1.0.16/CMakeLists.txt:31]: May be extract the common source and then add to LIBUSB_SOURCE platform-specific?

File: src/3rd_party-static/libusbx-1.0.16/msvc/errno.h [hunk 2]
```diff
@@ -0,0 +1,102 @@
+/* 
+ * errno.h
+ * This file has no copyright assigned and is placed in the Public Domain.
+ * This file is a part of the mingw-runtime package.
```
Comment (pestOO) [inline src/3rd_party-static/libusbx-1.0.16/msvc/errno.h:4]: Hm... I thought we had a problem with a mingw licence, didn't we?

Comment (anosach-luxoft) [inline src/3rd_party-static/libusbx-1.0.16/msvc/errno.h:4]: It's not my responsibility, I just use declared 3rd-party library

File: src/3rd_party-static/libusbx-1.0.16/CMakeLists.txt (hunk 1 above)
Comment (anosach-luxoft) [inline src/3rd_party-static/libusbx-1.0.16/CMakeLists.txt:31]: ok

**Comment by pestOO** [comment]
Reviewed

**Pull request merged**